devices/Mpu6886/Mpu6886.cs

```diff
@@ -0,0 +1,419 @@
+                registerAndOffset[1] = offsetbyte[0];
+                registerAndOffset[2] = offsetbyte[1];
+
+                BinaryPrimitives.WriteInt16BigEndian(offsetbyte, (short)value.Y);
+                registerAndOffset[3] = offsetbyte[0];
+                registerAndOffset[4] = offsetbyte[1];
+
+                BinaryPrimitives.WriteInt16BigEndian(offsetbyte, (short)value.Z);
+                registerAndOffset[5] = offsetbyte[0];
+                registerAndOffset[6] = offsetbyte[1];
+
+                _i2c.Write(registerAndOffset);
+            }
+        }
+
+        /// <summary>
+        /// Reset the internal registers and restores the default settings. (Datasheet, page 47)
+        /// </summary>
+        public void Reset()
+        {
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.PowerManagement1, 0b1000_0000 }));
+            Thread.Sleep(10);
+        }
+
+        /// <summary>
+        /// Set the chip to sleep mode. (Datasheet, page 47)
+        /// </summary>
+        public void Sleep()
+        {
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.PowerManagement1, 0b0100_0000 }));
+            Thread.Sleep(10);
+        }
+
+        /// <summary>
+        /// Disables the sleep mode. (Datasheet, page 47)
+        /// </summary>
+        public void WakeUp()
+        {
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.PowerManagement1, 0b0000_0000 }));
+            Thread.Sleep(10);
+        }
+
+        /// <summary>
+        /// Gets and sets the accelerometer full scale. (Datasheet page 37)
+        /// </summary>
+        public AccelerometerScale AccelerometerScale
+        {
+            get
+            {
+                SpanByte buffer = new byte[1];
+                Read(Mpu6886.Register.AccelerometerConfiguration1, buffer);
+                return (AccelerometerScale)buffer[0];
+            }
+
+            set
+            {
+                _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.AccelerometerConfiguration1, (byte)value }));
+                Thread.Sleep(1);
+            }
+        }
+
+        /// <summary>
+        /// Gets and sets the gyroscope full scale. (Datasheet page 37)
+        /// </summary>
+        public GyroscopeScale GyroscopeScale
+        {
+            get
+            {
+                SpanByte buffer = new byte[1];
+                Read(Mpu6886.Register.GyroscopeConfiguration, buffer);
+                return (GyroscopeScale)buffer[0];
+            }
+
+            set
+            {
+                _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.GyroscopeConfiguration, (byte)value }));
+                Thread.Sleep(1);
+            }
+        }
+
+
+        /// <summary>
+        /// Sets the enabled axes of the gyroscope and accelerometer. (Datasheet page 47)
+        /// </summary>
+        public EnabledAxis EnabledAxes
```

Comment:
Similar question as the previous one

Reply:
2 reserved (unused?) bits in the register, but added a mask just to on the safe side.